### PR TITLE
RDAPI-22833 - Add jersey level filter that reads rate limiting config and invalidates a session if the threshold is broken

### PR DESCRIPTION
**Pull request by credmond-axway**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by credmond-axway** [comment]
@JuanAxway Please review.

[Preview](https://deploy-preview-1642--axway-open-docs.netlify.app/docs/apim_administration/apimgr_admin/api_mgmt_config/#configure-the-api-manager-request-rate-limiter)

**Comment by JuanAxway** [comment]
@credmond-axway Can you please give me the preview link to see how it looks in the open docs?

**Comment by credmond-axway** [comment]
@JuanAxway I did. Its linked in my last message :)

**Review by JuanAxway — APPROVED** [review]

**Review by JuanAxway — APPROVED** [review]

File: content/en/docs/apim_administration/apimgr_admin/api_mgmt_config.md [hunk 1]
```diff
@@ -102,3 +97,23 @@ The following JVM setting is also required when importing the API in API Manage
...
+
+To configure the request rate limiter for your user' sessions in API Manager, perform the following steps in Policy Studio:
+
+1. Select **Environment Configuration > Listeners > API Gateway > API Portal > Paths** in the Policy Studio tree.
+2. Double-click the `API Portal v1.4 ('v1.4')` servlet to open its dialog box.
+3. On the servlet dialog box, click to **Edit** the `jersey.config.server.provider.classnames` property.
+4. Add `com.vordel.apiportal.api.filter.RateLimitBindingFeature` to the existing comma-separated list of class names.
+5. Click to **Add** two new properties to the **Servlet Properties** list :
+    * **Name**: `RateLimitFilter.rateLimitSize`. **Value**: Enter the number of requests, in milliseconds, that the user can send to API Manager.
+    * **Name**: `RateLimitFilter.rateLimitOffset`. **Value**: Enter the amount of time, in milliseconds, that the request rate limiter should allow between the user’s most recent request and the value set in `RateLimitFilter.rateLimitSize`).
```
Comment (andreamussap) [inline content/en/docs/apim_administration/apimgr_admin/api_mgmt_config.md:115]: @credmond-axway I've made changes for Axway style and improve readability. I'm just not sure about ("between the user’s most recent request and the value set in `RateLimitFilter.rateLimitSize`"), this is not clear to me. Can you please try to clarify this part? Thanks

https://deploy-preview-1642--axway-open-docs.netlify.app/docs/apim_administration/apimgr_admin/api_mgmt_config/#configure-api-manager-request-rate-limiter

cc @kmercierwhite

Comment (andreamussap) [reply content/en/docs/apim_administration/apimgr_admin/api_mgmt_config.md]: Thank you @credmond-axway and @jpfaffenzeller for feedback provided internally.

**Review by andreamussap — APPROVED** [review]

**Pull request merged**